scipy/stats/_distribution_infrastructure.py

```diff
@@ -3533,7 +3533,17 @@ def make_distribution(dist):
                 ``endpoints`` tuple above, and should define a subinterval of the
                 domain given by ``endpoints``.
 
-            A ``tuple`` value ``(a, b)`` is equivalent to ``{endpoints: (a, b)}``.
+            A ``tuple`` value ``(a, b)`` associated to a key in the ``parameters``
+            ``dict`` is equivalent to ``{endpoints: (a, b)}``.
```

Comment:
I think it might be easier to read with this as English; maybe elsewhere. 
```suggestion
            dictionary is equivalent to ``{endpoints: (a, b)}``.
```